src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
```

Comment:
`getRandomColor` method should return `String`

Reply:
I don't know why. Can you explain? Or should I ask it on QnA?